tools/explorer/lib/ontology_wrapper.py

```diff
@@ -42,7 +49,27 @@ def GetFieldsForTypeName(self,
     Returns:
             result_fields: a list of StandardField tuples
     """
-    pass
+    entity_type = self.universe.entity_type_universe.GetEntityType(
+        namespace,
+        entity_type_name
+    )
+    entity_type_fields = [
+        ETF(
+            namespace_name=field.field.namespace,
+            standard_field_name=field.field.field[0:],
```

Comment:
can this be simplified? 
`field.field.field`

Reply:
I dont think it can as I'm converting Optwrappers and FieldParts Named Tuples into Entity Type Field objects.

These named tuples are defined as such:
https://github.com/google/digitalbuildings/blob/6fa3ee2ecaa752e186173209272b7a93c917fc80/tools/validators/ontology_validator/yamlformat/validator/entity_type_lib.py#L34